<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackDialog</class>
 <widget class="QDialog" name="PlaybackDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playback</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QToolButton" name="rwdButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Slow down playback/acquisition</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/png/icons/gtk-media-forward-rtl.png</normaloff>:/new/png/icons/gtk-media-forward-rtl.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="stopButton">
              <property name="toolTip">
               <string>Stop playback/acquisition</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/png/icons/gtk-media-stop.png</normaloff>:/new/png/icons/gtk-media-stop.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="playButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/png/icons/gtk-media-play-ltr.png</normaloff>
                <normalon>:/new/png/icons/gtk-media-pause.png</normalon>:/new/png/icons/gtk-media-play-ltr.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="ffwdButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Speed up playback/acquisition</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/png/icons/gtk-media-forward-ltr.png</normaloff>:/new/png/icons/gtk-media-forward-ltr.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="snapshotButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/png/icons/snapshot.png</normaloff>:/new/png/icons/snapshot.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="recButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/png/icons/gtk-media-record.png</normaloff>:/new/png/icons/gtk-media-record.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="recTimedButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/png/icons/gtk-media-record-timed.png</normaloff>:/new/png/icons/gtk-media-record-timed.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="RecSettings">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/png/icons/gtk-properties.png</normaloff>:/new/png/icons/gtk-properties.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Delay (ms)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="fpsEdit">
                 <property name="toolTip">
                  <string extracomment="This is the time between successive images. If you want to give a fps value in Hz, then simply type &quot;1/fps&quot; and it will be transformed."/>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>100</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSlider">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>89</number>
                 </property>
                 <property name="value">
                  <number>19</number>
                 </property>
                 <property name="tracking">
                  <bool>true</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QToolButton" name="backButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/png/icons/gtk-go-back-ltr.png</normaloff>:/new/png/icons/gtk-go-back-ltr.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Rec Folder</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QComboBox" name="codecBox">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>0</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>FMF</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BUF-FMF</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HDF5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PNG</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BMP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MSMPEG4V2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>XVID</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>RAW</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>JPG</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="RecFolder">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QToolButton" name="toolButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/new/png/icons/gtk-open.png</normaloff>:/new/png/icons/gtk-open.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Rec Skipping</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="recSkippingEdit">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="LeftStatus">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="RightStatus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Framenumber</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>playButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>ffwdButton</tabstop>
  <tabstop>recButton</tabstop>
  <tabstop>RecSettings</tabstop>
  <tabstop>fpsEdit</tabstop>
  <tabstop>rwdButton</tabstop>
  <tabstop>backButton</tabstop>
  <tabstop>RecFolder</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>codecBox</tabstop>
  <tabstop>recTimedButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
